lightly/models/modules/__init__.py

```diff
@@ -31,15 +31,24 @@
 from lightly.utils import dependency as _dependency
 
 if _dependency.torchvision_vit_available():
+    # Requires torchvision >=0.12
     # Requires torchvision >=0.12
     from lightly.models.modules.masked_autoencoder import (
         MAEBackbone,
         MAEDecoder,
         MAEEncoder,
     )
+    from lightly.models.modules.masked_vision_transformer_torchvision import (
+        MaskedVisionTransformerTorchvision,
+    )
+
 if _dependency.timm_vit_available():
     # Requires timm >= 0.9.9
     from lightly.models.modules.heads_timm import AIMPredictionHead
+    from lightly.models.modules.masked_autoencoder_timm import MAEDecoder
```

Comment:
This doesn't work because this import now overwrites the `MAEDecoder` import from `lightly.models.modules.masked_autoencoder`.

Reply:
In `masked_autoencoder` we use Torchvision, whereas in `masked_autoencoder_timm` we use TIMM